pom.xml

```diff
@@ -784,6 +784,8 @@
                         <ENV_VAR_EXISTS_BOOLEAN>true</ENV_VAR_EXISTS_BOOLEAN>
                     </environmentVariables>
                     <trimStackTrace>false</trimStackTrace>
+                    <argLine>-Xmx1024m</argLine>
+                    <forkedProcessExitTimeoutInSeconds>120</forkedProcessExitTimeoutInSeconds>
```

Comment:
Why this this needed?

Reply:
`ClusterOperatorTest` is flaky and I was trying to find out, whether it is env issue or some bug in this tests. You can check the log for example there: https://dev.azure.com/cncf/strimzi/_build/results?buildId=40416&view=logs&j=66711cbf-ca69-5603-44b4-df4438498017&t=b249c5e2-84b8-5b78-6f19-55caa3f2c635